CMakeLists.txt

```diff
@@ -1054,7 +1044,6 @@ if(SWIFT_BUILD_STDLIB OR SWIFT_BUILD_SDK_OVERLAY)
   message(STATUS "  Leak Detection Checker Entrypoints: ${SWIFT_RUNTIME_ENABLE_LEAK_CHECKER}")
   message(STATUS "")
 
-  message(STATUS "Threading Package:                  ${SWIFT_THREADING_PACKAGE}")
```

Comment:
Please replace this info output elsewhere on a per-SDK basis, say [in `_report_sdk()`](https://github.com/apple/swift/blob/main/cmake/modules/SwiftConfigureSDK.cmake#L12) if possible.